api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -36,6 +36,8 @@
   private static final String CLUSTER_LABEL_AOU = "all-of-us";
   private static final String CLUSTER_LABEL_CREATED_BY = "created-by";
   private static final String WORKSPACE_CDR = "WORKSPACE_CDR";
```

Comment:
nit, and maybe a separate PR: I'd really like to pull out the environment variable names into an enum class, so we'd have `NotebookEnvironmentVariables.CDR_BQ_DATASET`. There's also a CDR_VERSION somewhere that could go in there. 

If you don't want to do that here I can do it in a separate PR.

Reply:
I'm going to say separate PR, because this is an existing problem. (I think it showed up in green because rebase.)